src/demo/share/jfc/CodePointIM/README.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <title>README - CodePointIM</title>
-  <meta http-equiv="content-type" content="text/html; charset=ISO-8859-1">
+  <meta http-equiv="content-type" content="text/html; charset=utf-8">
```

Comment:
I believe this line is equivalent to simple `<meta charset="utf-8"/>` which is widely used in java.desktop's docs.

Reply:
> That seems reasonable, yes. However, I'd like to keep this change to the absolute minimum. Feel free to open a separate issue for fixing this.

FIled: https://bugs.openjdk.org/browse/JDK-8356937